visualization/app/codeCharta/model/files/files.helper.spec.ts

```diff
@@ -90,9 +90,9 @@ describe("files", () => {
 		})
 
 		it("should return the first fileState found if multiple files match the fileName", () => {
-			files.concat([{ file: TEST_DELTA_MAP_A, selectedAs: FileSelectionState.None }])
+			const expectedFiles = [...files, { file: TEST_DELTA_MAP_A, selectedAs: FileSelectionState.None }]
 
-			const result = getFileByFileName(files, TEST_DELTA_MAP_A.fileMeta.fileName)
```

Comment:
This is an actual fix, the test should have failed without this change. I guess the test does not work as expected?

Reply:
Yes,I don't understand the behaviour either 
We should mark it to investigate